pkg/testutils/lint/lint_test.go

```diff
@@ -270,16 +267,8 @@ func TestLint(t *testing.T) {
 			}
 			data = data[0:n]
 
-			isApache := strings.HasPrefix(filename, "pkg/obsservice")
-			switch {
-			case isApache:
-				if apacheHeader.Find(data) == nil {
```

Comment:
This is the last use of the `apacheHeader` variable, so it can also be removed.

Reply:
done. PR up on `master` as well. I'll include it in the backports.